e2e_test/batch/basic/local/lookup_join.slt.part

```diff
@@ -29,12 +29,6 @@ select * from t1 join t3 on t1.v1 = t3.v1;
 ----
 1 2 1 2
```

Comment:
Is it possible to add more test cases to include more types?

Reply:
Do you mean more join types? Or more data types?